bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/template/factory/impl/flamingos2/Flamingos2Imaging.scala

```diff
@@ -38,6 +38,7 @@ case class Flamingos2Imaging(blueprint:SpFlamingos2BlueprintImaging) extends Fla
   val exposureTimes: Map[Flamingos2.Filter, Double] = {
     import Flamingos2.Filter._
     Map(H       -> 10.0,
+        K_LONG  -> 30.0,
         K_SHORT -> 30.0) withDefaultValue 60.0
   }
 
```

Comment:
I realize this is not your fault, but just to make me feel better can you change the declared type to `Flamingos2.Filter => Double`? It will still compile but will lose its mapness. Maps that have been crazified with `withDefaultValue` stop making sense.